theano/gpuarray/corr3d_gemm.c

```diff
@@ -201,6 +207,7 @@ KERNEL void dilated_col2im3d_kernel(const ga_size n,
 }
 
 #kernel col2im3d_kernel : size, *, size, size, size, size, size, size, size, size, size, size, size, size, size, size, size, size, size, *, size, size :
+
```

Comment:
I feel it's much easier to read when there is a space between the "header" and the function.  Why do you want to remove it?